libunicorn-sys/src/lib.rs

```diff
@@ -80,29 +78,24 @@ extern "C" {
 
 
 impl Error {
-    pub fn msg(&self) -> String {
-        error_msg(*self)
+    pub fn msg_str(self) -> &'static str {
+        unsafe {
+            let s = uc_strerror(self) as *const u8;
+            let mut p = s;
+            while 0 != *p { p = p.add(1); }
+            let l = p as usize - s as usize;
```

Comment:
Could you move these 3 lines to a function such as `cstr_len`, I think it make it easier to read.